Common/ItemSpawner.cs

```diff
@@ -2,40 +2,98 @@
 using System.Collections.Generic;
 using System.Linq;
 using PathOfTerraria.Common.Enums;
+using PathOfTerraria.Content.Items.Consumables.Maps;
+using PathOfTerraria.Content.Items.Currency;
+using PathOfTerraria.Content.Items.Gear;
 using Terraria.DataStructures;
 using Terraria.ID;
 
 namespace PathOfTerraria.Common;
 
 internal class ItemSpawner
 {
-	public static void SpawnRandomItem(Vector2 pos, int ilevel = 0, float dropRarityModifier = 0)
+	/// <summary>
+	/// Starts the drop process for a MobKill
+	/// </summary>
+	/// <param name="pos"></param>
```

Comment:
I would personally document these - including other parameters - as discerning their effect can be tough to guess, especially dropRarityModifier.

Reply:
Will make sure to adjust the docs when we get into the later phases of the drop table stuff